AWS SSO identity provider (#123)
common-fate/docs: docs/approvals/sso/aws-sso.mdx

<Tabs
  groupId="deployment-strategy"
  defaultValue="cfn"
  values={[
	  { label: "Cloudformation", value: "cfn" },
	  { label: "Policy as JSON", value: "json" },
	  ]}
>
<TabItem value="cfn">
<h4>Setup the Cloudformation Template</h4>
Run the following commands in your terminal to create `granted-identity-sso-role.yml`, a CloudFormation template which will be deployed to your AWS account. We recommend saving this alongside your granted-deployment.yml file in source control.

```bash
IDP_SYNC_ROLE_ARN=$(gdeploy output IDPSyncExecutionRoleARN)
API_ROLE_ARN=$(gdeploy output RestAPIExecutionRoleARN)

cat << EOF > granted-identity-sso-role.yml
Resources:
  GrantedIdentitySSORole:
    Type: AWS::IAM::Role
    Properties:
      AssumeRolePolicyDocument:
        Statement:
          - Action: sts:AssumeRole
            Effect: Allow
            Principal:
              AWS: "${IDP_SYNC_ROLE_ARN}"
          - Action: sts:AssumeRole
            Effect: Allow
            Principal:
              AWS: "${API_ROLE_ARN}"
        Version: "2012-10-17"

>       Description: This role grants management access to AWS SSO for the Granted Access Handler.

Reviewer: ```suggestion
      Description: This role grants read access to AWS SSO to Granted.
```
